demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -123,7 +124,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private final View.OnClickListener downloadBatchesOnClick = v -> {
         Batch batch = new Batch.Builder(BATCH_ID_1, "Made in chelsea")
-                .addFile(FILE_ID_1, FIVE_MB_FILE_URL)
+                .addFile(FILE_ID_1, FIVE_MB_FILE_URL, FilePathCreator.create("/foo/bar/10mb.zip"))
```

Comment:
Looking at this I'm wondering... does the client still care about the file ID at this point?

Reply:
They might not care about the file id. It's an optional really but I would rather keep it as is until I can find time to create a more fluent builder. For example I'd rather we had something like:

```
Batch.Builder(BATCH_ID_1, "Made in chelsea")
         .addFile(URL).withId(FILE_ID).done()
         .addFile(ADDITIONAL_URL).withStoragePath(PATH).done()
.build()
```

Reply:
Ok, let's keep it for now, it's optional you are right...
I like that builder example you showed 💯  (I think it could work even without the `done()` part).
Not for this PR though